index.html

```diff
@@ -20,6 +20,7 @@
     </style>
     <link rel="stylesheet" href="assets/css/bootstrap-theme.min.css">
     <link rel="stylesheet" href="assets/css/main.css">
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
```

Comment:
No need to create more style sheets. This change can be done in `main.css` which are our own custom changes.

Please remove this and move the corresponding media query in main.css at the bottom.